backend/routes_chemicals.py

```
@@ -1,32 +1,17 @@
-from flask import request, jsonify, session
+from flask import request, jsonify
 from models import db, Chemical, ChemicalIssuance, User, AuditLog, UserActivity
 from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)